### PR TITLE
fix-point3

### PR DESCRIPTION
**Pull request by Full-Time117**
react-template-6 (starbucks template) fix point3-  add address map bug fix, change order type bug fix, improve styles most similar design

### CODE REVIEW

**Review by carlosq-ordering — APPROVED** [review]

File: src/themes/six/src/components/LoginForm/index.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ const LoginFormUI = (props) => {
   const [willVerifyOtpState, setWillVerifyOtpState] = useState(false)
   const [validPhoneFieldState, setValidPhoneField] = useState(false)
   const [otpState, setOtpState] = useState('')
-  const [otpLeftTime, _, resetOtpLeftTime] = useCountdownTimer(
+  const [otpLeftTime, resetOtpLeftTime] = useCountdownTimer(
```
Comment (YisusJe) [inline src/themes/six/src/components/LoginForm/index.js:77]: resetOtpLeftTime comes in the third parameter of the hook. Revert this change or send a initialOtpTime for each resetOtpLeftTime in the code

**Review by YisusJe — COMMENTED** [review]
Review changes

File: src/themes/six/src/components/LoginForm/index.js (hunk 1 above)
Comment (Full-Time117) [reply src/themes/six/src/components/LoginForm/index.js:77]: Ok I will keep third parameter.

**Review by ramonOrdering — COMMENTED** [review]
ok
